src/megacmd.cpp

```diff
@@ -2025,7 +2025,7 @@ string getHelpStr(const char *command)
         os << " -h" << "\t" << "Human readable" << endl;
         os << " --versions" << "\t" << "Calculate size including all versions." << endl;
         os << "   " << "\t" << "You can remove all versions with \"deleteversions\" and list them with \"ls --versions\"" << endl;
-        os << " --path-display-size=N" << "\t" << "Use a fixed size of N characters for paths" << endl;
+        os << " --path-display-size=N" << "\t" << "Use at least N characters for displaying paths" << endl;
```

Comment:
`du` command does not honor the "at least" N characters, as does not use `ColumnDisplayer`. Same for  `backup` and `mediainfo`commands.